BazyDanych/BazaDanych/BazaDanych/Model/Woman.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BazaDanych.Model
+{
+    internal class Woman
+    {
+        public string WomanName { get; set; }
```

Comment:
FIX: Tutaj w ogóle nie ma konstruktora. Jak robisz we wszystkich modelach to tutaj też zrób.

Reply:
[8a9fa36](https://github.com/broniuu/Praktyki2022/pull/3/commits/8a9fa36411257f7a82d506287f7faaab771bddd6)